test/parallel/test-quic-simple.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const fixtures = require('../common/fixtures');
+const key = fixtures.readKey('agent8-key.pem', 'binary');
+const cert = fixtures.readKey('agent8-cert.pem', 'binary');
+const ca = fixtures.readKey('fake-startcom-root-cert.pem', 'binary');
+
+const createSocket = require('quic');
+
+const socket = createSocket({ type: 'udp4', port: 1234 });
+
+socket.listen({ key, cert, ca });
+
+socket.on('session', common.mustCall((session) => {
+  console.log(session);
+
+  session.on('secure', () => {
+    // We can only open a stream after the handshake has completed
+    const uni = session.openUnidirectionalStream();
+    uni.write("I wonder if it worked.");
+    uni.end('test');
+  });
```

Comment:
From an API perspective, we should not wait for this event. In other terms, the `'secure'` event is not providing any additional behavior for a user.

Reply:
Yeah, this is temporary until we get the signaling and state mechanism fully implemented inside the session and stream objects.